src/api-docs.js

```diff
@@ -591,6 +595,458 @@ const apiDoc = {
+                    summary: 'Wallet id parameter is invalid',
+                    value: { success: false, message: 'Invalid wallet-id parameter.' }
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+    '/wallet/tx-proposal/add-signatures': {
+      post: {
+        summary: 'Add signatures to the transaction and return the txHex with the signatures.',
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            in: 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          },
+        ],
+        requestBody: {
+          description: 'Transaction hex and signatures',
+          required: true,
+          content: {
+            'application/json': {
+              schema: {
+                type: 'object',
+                required: ['txHex', 'signatures'],
+                properties: {
+                  txHex: {
+                    type: 'string',
+                    description: 'Transaction hex representation.'
+                  },
+                  signatures: {
+                    type: 'array',
+                    description: 'Signatures collected for the transaction.',
+                    items: {
+                      type: 'object',
+                      required: ['index', 'data'],
+                      properties: {
+                        index: {
+                          type: 'number',
+                          description: 'Input index this signature refers to.',
+                        },
+                        data: {
+                          type: 'string',
+                          description: 'Hex representation of the input data to be added on the transaction.',
+                        },
+                      },
+                    }
+                  },
+                }
+              },
+            }
+          }
+        },
+        responses: {
+          200: {
+            description: 'Get transaction hex with input data.',
+            content: {
+              'application/json': {
+                examples: {
+                  success: {
+                    summary: 'Success',
+                    value: { success: true, txHex: '0123abc...' }
+                  },
+                  'no-wallet-id': {
+                    summary: 'No wallet id parameter',
+                    value: { success: false, message: "Parameter 'wallet-id' is required." }
+                  },
+                  'invalid-wallet-id': {
+                    summary: 'Wallet id parameter is invalid',
+                    value: { success: false, message: 'Invalid wallet-id parameter.' }
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+    '/wallet/tx-proposal/push-tx': {
```

Comment:
I was thinking about this API here... semantically we could change this to be only `push-tx`. We don't even need a wallet to push the tx, given that we have the hex, right? So it shouldn't be inside the `tx-proposal` group of APIs and not even the `wallet` group of APIs. What do you think?